src/utilities/theme/role-variants.ts

```diff
@@ -1,19 +1,17 @@
 import {RoleVariants} from './types';
 
-// eslint-disable-next-line shopify/typescript/prefer-pascal-case-enums
-export enum UNSTABLE_Color {
-  Surface = '#FAFAFA',
-  DarkSurface = '#111213',
-  OnSurface = '#111213',
-  Interactive = '#2E72D2',
-  Neutral = '#111213',
-  Primary = '#008060',
-  Critical = '#D82C0D',
-  Warning = '#FFC453',
-  Highlight = '#5BCDDA',
-  Success = '#008060',
-  Decorative = '#FDC7CE',
-}
+export const DEFAULT_COLOR = {
```

Comment:
Slightly tangental but I was chatting with Tim yesterday, what do you think about naming this "list of colors" a Palette?

So this becomes DEFAULT_PALETTE, and buildColors becomes buildPalette.

Reply:
Probably should be called theme rather than palette (referred to this in another comment). And pascal case for enums? `DefaultTheme` (even though the keys are not pascal case)

Reply:
And the factor really is a color role variant factory, but that’s a long name :)

Reply:
`DefaultTheme` it is